class.jetpack-gutenberg.php

```diff
@@ -1,78 +1,71 @@
 <?php
+function jetpack_register_block( $type, $args = array() ) {
```

Comment:
Could you add a docblock for that function? I suspect we'll get to use that function a lot in the future, so it would be useful and helpful if it was fully documented so our IDEs can use that information.

I would recommend doing the same with other functions and variable declarations in this file to make our linters happy.